### ISSUE TITLE
[github_actions] steam-native-runtime does not install

### ISSUE THREAD

**Issue by LukeShortCloud** [LukeShortCloud/winesapOS#806]
The package `libibus` consistently fails to install and says `Write failed` on each run of our GitHub Actions workflow now. This is installed alongside `steam-native-runtime` which causes it to not be installed since one of the dependencies had an issue.

```
installing libibus...
error: could not extract /usr/share/ibus/dicts/emoji-zh_Hant.dict (Write failed)
error: could not extract /usr/share/ibus/dicts/emoji-zh_Hant_HK.dict (Write failed)
error: could not extract /usr/share/ibus/dicts/emoji-zu.dict (Write failed)
error: could not extract /usr/share/ibus/dicts/unicode-names.dict (Write failed)
error: could not extract /usr/share/vala/vapi/ibus-1.0.vapi (Write failed)
error: problem occurred while installing libibus
error: could not commit transaction
error: failed to commit transaction (transaction aborted)
Errors occurred, no packages were upgraded.
```

https://github.com/LukeShortCloud/winesapOS/actions/runs/8491230799/job/23263026063
https://github.com/LukeShortCloud/winesapOS/actions/runs/8489012006/job/23258641853

**Comment by LukeShortCloud** [comment]
This is definitely something related to storage getting full on the `winesapos.img` file. The odd thing is that it reports that there is 5.5 GiB of space in total and that only 5.0 GiB is used. However, 19 MiB is the available space left. This probably has to do with Btrfs as some files could be using copy-on-write. We need to get storage space information from Btrfs which will be more accurate.

```
Viewing final storage space usage...
+ df -h
Filesystem                   Size  Used Avail Use% Mounted on
overlay                       73G   64G  8.8G  88% /
tmpfs                        7.9G     0  7.9G   0% /sys/fs/cgroup
/dev/mapper/buildvg-buildlv   94G  6.3G   88G   7% /workdir
devtmpfs                     7.9G     0  7.9G   0% /dev
tmpfs                        7.9G  4.0K  7.9G   1% /dev/shm
/dev/root                     73G   64G  8.8G  88% /etc/hosts
/dev/loop5p4                 5.5G  5.0G   19M 100% /winesapos
/dev/loop5p4                 5.5G  5.0G   19M 100% /winesapos/home
/dev/loop5p4                 5.5G  5.0G   19M 100% /winesapos/swap
/dev/loop5p3                 974M  311M  597M  35% /winesapos/boot
/dev/loop5p2                 510M  160K  510M   1% /winesapos/boot/efi
tmpfs                        7.9G     0  7.9G   0% /winesapos/tmp
tmpfs                        7.9G  252K  7.9G   1% /winesapos/var/log
tmpfs                        7.9G     0  7.9G   0% /winesapos/var/tmp
```

**Comment by LukeShortCloud** [comment]
Confirmed that about 500 MiB is being used by duplicate data.

```
+ btrfs filesystem df -h /winesapos
Data, single: total=4.52GiB, used=4.50GiB
System, DUP: total=8.00MiB, used=16.00KiB
Metadata, DUP: total=493.00MiB, used=209.58MiB
GlobalReserve, single: total=16.75MiB, used=0.00B
```

https://github.com/LukeShortCloud/winesapOS/actions/runs/8491526545/job/23263581412

**Comment by LukeShortCloud** [comment]
I found the problem. KDE Plasma 6 is taking up a lot more space than 5. That is because not everything has been ported over so both the Qt 5 and the new Qt 6 libraries and old dependencies are installed. This was originally brought up and addressed here: #779

I did not bump up the size of the image used by CI at the time as it did not appear to need it (it just barely had enough space before). Now it should be increased from 7 to 8 GiB to work properly.

**Comment by LukeShortCloud** [comment]
Increasing the raw image size fixed the issue.

```
2024-03-30T16:47:47.0147912Z + for i in ${@}
2024-03-30T16:47:47.0148334Z + echo -n '\tsteam-native-runtime...'
2024-03-30T16:47:47.0149280Z \tsteam-native-runtime...+ pacman_search steam-native-runtime
2024-03-30T16:47:47.0150078Z + chroot /winesapos pacman -Qsq steam-native-runtime
2024-03-30T16:47:47.0373688Z + '[' 0 -eq 0 ']'
2024-03-30T16:47:47.0374110Z + echo PASS
2024-03-30T16:47:47.0374326Z PASS
```

```
2024-03-30T16:47:49.0926821Z Viewing final storage space usage...
2024-03-30T16:47:49.0927152Z + df -h
2024-03-30T16:47:49.0964077Z Filesystem                   Size  Used Avail Use% Mounted on
2024-03-30T16:47:49.0965054Z overlay                       73G   64G  8.8G  88% /
2024-03-30T16:47:49.0966091Z tmpfs                        7.9G     0  7.9G   0% /sys/fs/cgroup
2024-03-30T16:47:49.0969728Z /dev/mapper/buildvg-buildlv   94G  6.3G   88G   7% /workdir
2024-03-30T16:47:49.0970528Z devtmpfs                     7.9G     0  7.9G   0% /dev
2024-03-30T16:47:49.0971238Z tmpfs                        7.9G  4.0K  7.9G   1% /dev/shm
2024-03-30T16:47:49.0971951Z /dev/root                     73G   64G  8.8G  88% /etc/hosts
2024-03-30T16:47:49.0972931Z /dev/loop5p4                 6.5G  5.0G 1009M  84% /winesapos
2024-03-30T16:47:49.0973743Z /dev/loop5p4                 6.5G  5.0G 1009M  84% /winesapos/home
2024-03-30T16:47:49.0974581Z /dev/loop5p4                 6.5G  5.0G 1009M  84% /winesapos/swap
2024-03-30T16:47:49.0975399Z /dev/loop5p3                 974M  311M  597M  35% /winesapos/boot
2024-03-30T16:47:49.0976265Z /dev/loop5p2                 510M  160K  510M   1% /winesapos/boot/efi
2024-03-30T16:47:49.0977092Z tmpfs                        7.9G     0  7.9G   0% /winesapos/tmp
2024-03-30T16:47:49.0977849Z tmpfs                        7.9G  252K  7.9G   1% /winesapos/var/log
2024-03-30T16:47:49.0978650Z tmpfs                        7.9G     0  7.9G   0% /winesapos/var/tmp
2024-03-30T16:47:49.0979731Z + btrfs filesystem df -h /winesapos
2024-03-30T16:47:49.0980520Z Data, single: total=5.48GiB, used=4.50GiB
2024-03-30T16:47:49.0980962Z System, DUP: total=8.00MiB, used=16.00KiB
2024-03-30T16:47:49.0981496Z Metadata, DUP: total=512.00MiB, used=208.86MiB
```

https://github.com/LukeShortCloud/winesapOS/actions/runs/8491940824/job/23264424478
